curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.english.md

```diff
@@ -47,13 +55,7 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var ourStr = "I come first. " + "I come second.";
-
-// Only change code below this line
-
-var myStr;
-
+var myStr; // Change this line
```

Comment:
```suggestion
var myStr; // Only change this line
```

Reply:
I was hoping to avoid having an `Only change this line`.  I will have to check if I added the `Only change code below this line` and `Only change code above this line` in a separate PR for this challenge.